src/js/main.js

```diff
@@ -688,13 +716,15 @@ Site.Minimap = {
     var y = row * Site.Map.window.height * -1;
 
     // Move map to new coordinates
-    // moveIndicator is called by this function
     Site.Map.moveMap(x,y);
   },
 
-  moveIndicator: function(x,y) {
+  moveIndicator: function(event) {
```

Comment:
I think you should still parse the `x, y` here so you can use this function in other places if needed